molecule/default/verify.yml

```
@@ -28,3 +28,33 @@
         - gitlab_edition is defined
         - gitlab_version is defined
         - gitlab_release is defined
+
+    - name: "Check GitLab Health endpoint"
+      ansible.builtin.uri:
+        url: "http://localhost/-/health"
+      register: "health_check"
+      failed_when: "'OK' not in health_check.msg or health_check.status!=200"
+
+    - name: "Check GitLab Readiness endpoint"
+      ansible.builtin.uri:
+        url: "http://localhost/-/readiness?all=1"
+      register: "readiness_check"
+      failed_when: "readiness_check.status == 503"
+
+    - name: "Check GitLab Liveness endpoint"
+      ansible.builtin.uri:
+        url: "http://localhost/-/liveness"
+      register: "liveness_check"
+      failed_when: "liveness_check.status == 503"
```

Do you think this more explicit check of the exit code could be beneficial although it is of course implicitly checked in the health check?

```suggestion
      failed_when: "liveness_check.rc != 0 or liveness_check.status == 503"
```

---

That does not work